code/game/machinery/civilian_bounties.dm

```diff
@@ -247,10 +244,100 @@
 	icon_state = "bounty_cube"
 	///Value of the bounty that this bounty cube sells for.
 	var/bounty_value = 0
+	///Multiplier for the bounty payout received by the Supply budget if the cube is sent without having to nag.
+	var/speed_bonus = 0.2
+	///Multiplier for the bounty payout received by the person who completed the bounty.
+	var/holder_cut = 0.3
+	///Multiplier for the bounty payout received by the person who claims the handling tip.
+	var/handler_tip = 0.1
+	///Time until speed bonus expires.
+	var/bonus_countdown = 5 MINUTES
```

Comment:
I'm confused why this is a var, given that the speed bonus is lost when the cube first nags.